cmd/vault-manager/main.go

```diff
@@ -162,7 +161,10 @@ func main() {
 		}
 
 		if runOnce {
-			return
+			if erred {
+				os.Exit(1)
```

Comment:
A fun trivia here. If you were to keep the counter, then you could reach out for some bit twiddling magic:

```
os.Exit((^counter + 1) >> 31 & 1)
```

This way, any non-zero value would exit with code 1, and zero would be 0.